pages/common/routers/form.js

```diff
@@ -204,6 +234,9 @@ module.exports = ({
   };
 
   const _process = (req, res, next) => {
+    // if we don't clone the schema here it gets modified in place by flattenDetailsReveals() because object refs...
+    // thought it would be clearer with an explicit assignment
+    req.form.schema = flattenDetailsReveals(cloneDeep(req.form.schema));
```

Comment:
It's probably safer to do this inside the function

Reply:
its recursive tho...